sdk/storage/Azure.Storage.Blobs.Batching/src/DelayedResponse.cs

```diff
@@ -77,6 +77,11 @@ public void SetLiveResponse(Response live, bool throwOnFailure)
             }
         }
 
+        public override string ToString()
+        {
+            return _live == null ? "Status: NotSent, Request is not sent yet." : base.ToString();
```

Comment:
Nit - can we change this to `Status: NotSent, the batch has not been submitted yet.` to be super explicit if someone sees this while debugging?

Reply:
Done